src/AutorestSwift/View Models/OperationViewModel.swift

```diff
@@ -140,8 +142,18 @@ struct OperationViewModel {
             responses.append(ResponseViewModel(from: response, with: operation))
         }
 
+        var defaultException: ExceptionResponseViewModel?
         for exception in operation.exceptions ?? [] {
-            exceptions.append(ExceptionResponseViewModel(from: exception))
+            let vm = ExceptionResponseViewModel(from: exception)
```

Comment:
I would avoid using `vm`. In Azure-speak it means "virtual machine" and Swift frowns on variables with fewer than 3 characters anyways.

Reply:
Renamed.